lib/Backends/CPU/CPUBackend.cpp

```diff
@@ -409,6 +409,14 @@ bool CPUBackend::shouldLower(const Node *N) const {
   }
 }
 
+unsigned CPUBackend::numDevices() {
+#if defined(GLOW_WITH_CPU)
+  return std::thread::hardware_concurrency();
+#else
+  return 0;
```

Comment:
Don't think this is necessary anymore. If we don't build with `GLOW_WITH_CPU` then the CPU would never be registered with the factory anyway. Same on the other backends too.